src/mixins/ReactMixin/util/buildComponents.js

```diff
@@ -21,14 +30,15 @@ const buildComponents = (
         return undefined
       }
 
-      const result = builder.apply(view)
+      const result = typeof builder === 'function'

```

Comment:
This looks slightly fishy :) What other type can the `builder` have?

Reply:
The API is very flexible to accommodate a variety of situations. The builder can actually be a function, a component (instance or class) or an object literal. Perhaps I should add an abstraction here.

Reply:
Updated!